frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -40,14 +40,11 @@ export default function ViewFacility() {
     }
   }, [])
 
-  const MyBookingButton = (
-    <Button
-      state="primary"
-      text="Bookings"
-      type="button"
-      size="small"
+  const myBookingsIcon = (
+    <img
+      src={MyBookingsIcon}
       onClick={() => {
-        history.push(`${PATHS.VIEW_MY_BOOKINGS_USERID}/${localStorage.getItem('userID')}`)
+        history.push(PATHS.VIEW_MY_BOOKINGS_USERID + '/' + localStorage.getItem('userID'))
```

Comment:
why was this updated?

Reply:
eh what the. i didnt touch this. okay i change back